testkit/src/integration-testkit.ts

```diff
@@ -175,7 +175,7 @@ export class IntegrationTestkit {
         'VERSION_CHECK_ON_STARTUP',
         process.env.VERSION_CHECK_ON_STARTUP || 'true',
       )
-      .withWaitStrategy(Wait.forLogMessage('Starting Kalix Proxy'))
+      .withWaitStrategy(Wait.forLogMessage('Kalix Runtime started'))
```

Comment:
This did expect "Starting" before (even that message has changed in the Runtime), but this changes it to await the "started" state as we do in the JVM SDK testkit, as well.